### PR TITLE
support adding and displaying dishes on BE

### PR DESCRIPTION
**Pull request by Isterius**

### CODE REVIEW

**Review by pawelkondraciuk — CHANGES_REQUESTED** [review]

File: apps/nestjs-api/src/app/entities/dish.entity.ts [hunk 1]
```diff
@@ -26,6 +28,13 @@ export class DishEntity {
   })
   order: OrderEntity;
 
+  @OneToOne(() => UserEntity)
```
[the remark this answers is not in the document]
Comment (Isterius) [reply apps/nestjs-api/src/app/entities/dish.entity.ts:31]: for dish it is person who added it

File: apps/nestjs-api/src/app/entities/dish.entity.ts [hunk 2]
```diff
@@ -26,6 +28,13 @@ export class DishEntity {
...
+  @JoinColumn()
+  orderer: UserEntity
+
+  @Column({ default: false })
+  paid: boolean;
```
[the remark this answers is not in the document]
Comment (Isterius) [reply apps/nestjs-api/src/app/entities/dish.entity.ts:36]: how we can sum balance for user if we don't know if he paid for specific dish?

File: libs/api/data-transfer/src/lib/create-dish-body.model.ts [hunk 3]
```diff
@@ -0,0 +1,6 @@
+export interface CreateDishBody {
```
[the remark this answers is not in the document]
Comment (Isterius) [reply libs/api/data-transfer/src/lib/create-dish-body.model.ts:1]: true, I wanted to remove one :D thx

File: apps/nestjs-api/src/app/entities/dish.entity.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (pawelkondraciuk) [reply apps/nestjs-api/src/app/entities/dish.entity.ts:31]: We could add dish owner to UserDish

File: apps/nestjs-api/src/app/entities/dish.entity.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (pawelkondraciuk) [reply apps/nestjs-api/src/app/entities/dish.entity.ts:36]: Do we need to know if he paid for that dish? We could just sum dish price and shipping and add it to user balance

File: apps/nestjs-api/src/app/entities/dish.entity.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (pawelkondraciuk) [reply apps/nestjs-api/src/app/entities/dish.entity.ts:31]: But I think there won't be any special powers for dish owner

[the remark this answers is not in the document]
Comment (Isterius) [reply apps/nestjs-api/src/app/entities/dish.entity.ts:31]: for now it's only needed to show `delete`/`ediit` controls in dish row

[the remark this answers is not in the document]
Comment (pawelkondraciuk) [reply apps/nestjs-api/src/app/entities/dish.entity.ts:31]: Can we use user related to the dish via UserDishEntity? There is already dishOwner field

[the remark this answers is not in the document]
Comment (Isterius) [reply apps/nestjs-api/src/app/entities/dish.entity.ts:31]: 👀 🔍

**Pull request closed without merging**